spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/hive/HivePropertiesConverter.java

```diff
@@ -6,15 +6,67 @@
 package com.datastrato.gravitino.spark.connector.hive;
 
 import com.datastrato.gravitino.spark.connector.PropertiesConverter;
+import com.google.common.collect.ImmutableMap;
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
+import java.util.Optional;
+import javax.ws.rs.NotSupportedException;
 
-/** Transform hive catalog properties between Spark and Gravitino. Will implement in another PR. */
+/** Transform hive catalog properties between Spark and Gravitino. */
 public class HivePropertiesConverter implements PropertiesConverter {
 
+  // Transform Spark format to Gravitino format
+  static final Map<String, String> hiveTableFormatMap =
+      ImmutableMap.of(
+          "sequencefile", "SEQUENCEFILE",
+          "rcfile", "RCFILE",
+          "orc", "ORC",
+          "parquet", "PARQUET",
+          "textfile", "TEXTFILE",
+          "avro", "AVRO");
+
+  static final Map<String, String> sparkToGravitinoPropertyMap =
+      ImmutableMap.of(
+          "hive.output-format",
+          HivePropertyConstants.GRAVITINO_HIVE_OUTPUT_FORMAT,
+          "hive.input-format",
+          HivePropertyConstants.GRAVITINO_HIVE_INPUT_FORMAT,
+          "hive.serde",
+          HivePropertyConstants.GRAVITINO_HIVE_SERDE_LIB);
+
```

Comment:
Can we use the definition of properties in the class 'com.datastrato.gravitino.trino.connector.catalog.hive.HivePropertyMeta' in the gravitino-bundled-catalog?

Reply:
yes, we'd better use `gravitino-bundled-catalog`, For now I'm not using it because I'm confusing why `gravitino-bundled-catalog` depends are GravitinoXXCatalogs packages, I don't want to introduce so many gravitino catalog  dependces to spark connector. cc @yuqi1129

Reply:
`gravitino-bundled-catalog` is a project that integrated catalogs of commonly used properties and used to expose to engines like Trino and Spark. Consider the scenario: If we change the property key in the hive catalog how can you ensure that all execution engines that use that catalog can change the property accordingly?

Reply:
I know what problems `gravitino-bundled-catalog` is used to resolve. My concern is, it introduces many catalog-related packages, Ideally, I think the catalog properties are defined in `gravitino-bundled-catalog`,  spark,trino and xxCatalog  all depend on bundled-catalog. cc @jerryshao WDYT?
```
dependencies {
// will introduce more and more catalog packages.
  implementation(project(":catalogs:catalog-hive"))
  implementation(project(":catalogs:catalog-jdbc-common"))
  implementation(project(":catalogs:catalog-jdbc-mysql"))
  implementation(project(":catalogs:catalog-jdbc-postgresql"))
  implementation(project(":catalogs:catalog-lakehouse-iceberg"))
}
```

Reply:
I also believe that you should use `gravitino-bundled-catalog` because `gravitino-bundled-catalog` is designed for this scenario. If it cannot be used, then `gravitino-bundled-catalog` should be modified.

BTW, you are using the shadow jar, the dependencies were excluded

Reply:
using properties defined in `gravitino-bundled-catalog`,  I put all properties from `gravitino-bundled-catalog` in `HivePropertiesConstant`.